Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md

```diff
@@ -18,6 +18,9 @@ Alerts v2:
 1. SecurityAlert.Read.All - Application (required for the commands: `msg-search-alerts` and `msg-get-alert-details`)
 2. SecurityAlert.ReadWrite.All - Application (required for updating alerts with the commands: `msg-update-alert` and `msg-create-alert-comment`)
 
+    #### Note
+    When using Alerts V2: The following properties support $filter : assignedTo, classification, determination, createdDateTime, lastUpdateDateTime, severity, serviceSource and status.
```

Comment:
I merged it with the first Note (need to delete the first)
```suggestion
#### Note
- The `message-search-alerts` command does not filter alerts of the `Office 365` provider because of API limitations.\
For more info, see: https://github.com/microsoftgraph/security-api-solutions/issues/56.
- When using Alerts V2: only the following properties are supported as filters on the *Fetched incidents filter* parameter and *filter* argument: assignedTo, classification, determination, createdDateTime, lastUpdateDateTime, severity, serviceSource and status. As per [Microsoft's documentation](https://learn.microsoft.com/en-us/graph/api/security-list-alerts_v2?view=graph-rest-1.0&tabs=http#optional-query-parameters).
```
wdyt?